### ISSUE TITLE
Manual method documentation conflicts with Raku declarators (via Pod::To::Markdown)

### ISSUE THREAD

**Issue by dwarring** [libxml-raku/LibXML-raku#34]
Manual method documentation style is typically:
```
=begin item
my-method

description here
=end item
```
Which produces markdown:
```
### my-method

description here
```

Compare this with  declarator blocks in source code:
```
#| description here
method my-method() { ... }
```
Which produces markdown:
```
method my-method
-----

Description here.
```
 I'm not sure now where the former convention came from. But it conflicts and make it difficult to mix the two.

I suggest following the latter convention or switch to using Pod declarators

**Comment by dwarring** [comment]
Going down the path of using Pod decorators. Have done LibXML::Document as an example. Others will follow.

**Comment by dwarring** [comment]
Largely done. Included in LibXML 0.4.0 release
